lib/puppet-lint/plugins/check_whitespace/left_lbrace_whitespace.rb

```diff
@@ -0,0 +1,24 @@
+# Public: Check the manifest tokens for whitespace before left bracket that is missing and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
+PuppetLint.new_check(:left_lbrace_whitespace) do
```

Comment:
Is there any situation where the left curly brace should not have whitespace between it and the previous token? I didn't want to restrict it to resource types, only, but if we're better off being pedantic then I will do that.